src/polkadot/fellows.ts

```diff
@@ -10,83 +15,121 @@ export class PolkadotFellows implements TeamApi {
-
-      this.logger.debug("Connecting to relay parachain.");
-      // We connect to the relay chain
-      api = await ApiPromise.create({ provider: new WsProvider("wss://rpc.polkadot.io") });
-
-      // We iterate over the different members and extract their data
-      const users: Map<string, number> = new Map<string, number>();
-      for (const fellow of fellows) {
-        this.logger.debug(`Fetching identity of '${fellow.address}', rank: ${fellow.rank}`);
-        const identityQuery = await api.query.identity.identityOf(fellow.address);
-        // If the identity is null, we check if there is a super identity.
-        if (identityQuery.isEmpty) {
-          this.logger.debug("Identity is null. Checking for super identity");
-          const superIdentity = (await api.query.identity.superOf(fellow.address)).toHuman() as
-            | [string, { Raw: string }]
-            | undefined;
-          if (superIdentity && superIdentity[0]) {
-            this.logger.debug(`${fellow.address} has a super identity: ${superIdentity[0]}. Adding it to the array`);
-            fellows.push({ address: superIdentity[0], rank: fellow.rank });
+      // Create smoldot chain with Polkadot Relay Chain
+      const smoldotRelayChain = await smoldot.addChain({
+        chainSpec: polkadotChainSpec,
+      });
+
+      // Add the people chain to smoldot
+      const peopleRelayChain = await smoldot.addChain({
+        chainSpec: peopleChainSpec,
+        potentialRelayChains: [smoldotRelayChain],
+      });
+
+      // Initialize the smoldot provider
+      const jsonRpcProvider = getSmProvider(peopleRelayChain);
+      logger.info("Initializing the people client");
+      const peopleClient = createClient(jsonRpcProvider);
+
+      // Get the types for the people client
+      const peopleApi = peopleClient.getTypedApi(people);
+
+      const getGhHandle = async (address: SS58String): Promise<string | undefined> => {
+        logger.debug(`Fetching identity of '${address}'`);
+        const identityOf = await peopleApi.query.Identity.IdentityOf.getValue(address);
+
+        if (identityOf) {
+          const [identity] = identityOf;
+          const github = identity.info.github.value;
+
+          if (!github) {
+            logger.debug(`'${address}' does not have an additional field named 'github'`);
+            return;
+          }
+
+          const handle = github.asText().replace("@", "") as string;
+
+          if (handle) {
+            logger.info(`Found github handle for '${address}': '${handle}'`);
           } else {
-            this.logger.debug("No super identity found. Skipping");
+            logger.debug(`'${address}' does not have a GitHub handle`);
+            return;
           }
-          continue;
+          return handle;
         }
 
-        const [fellowData] = identityQuery.toHuman() as [Record<string, unknown>, unknown];
+        logger.debug(`Identity of '${address}' is null. Checking for super identity`);
```

Comment:
A leftover? Doesn't look like we've queried anything yet

Reply:
No, we are looking for the user's github address in the super identity if they have one.

It was implemented in #107